arch/src/x86_64/mod.rs

```diff
@@ -718,6 +723,38 @@ pub fn generate_common_cpuid(
         });
     }
 
+    // mwait
+    // Intel-defined CPU features, CPUID level 0x00000001 (ECX)
```

Comment:
I believe you mean "CPUID leaf" instead of "CPUID level".

Reply:
Thanks, I will modify it soon.